static/src/javascripts/projects/common/modules/experiments/tests/adblock-ask.js

```diff
@@ -7,23 +7,25 @@ import config from 'lib/config';
 const supportUrl = `${supportContributeURL()}?acquisitionData=%7B%22componentType%22%3A%22ACQUISITIONS_OTHER%22%2C%22source%22%3A%22GUARDIAN_WEB%22%2C%22campaignCode%22%3A%22shady_pie_open_2019%22%2C%22componentId%22%3A%22shady_pie_open_2019%22%7D&INTCMP=shady_pie_open_2019`;
 
 const askHtml = `
-<div class="contributions__adblock--moment">
-    <div class="contributions__adblock--moment-content">
-        <div class="contributions__adblock--moment-header">
-            <h2 class="contributions__adblock--moment-header--blue">Support</h2>
-            <h2 class="contributions__adblock--moment-header--blue">The Guardian’s</h2>
-            <h2 class="contributions__adblock--moment-header--orange">model for open, independent journalism</h2>
+<div class="contributions__adblock">
```

Comment:
I've seen ad blocker rules that block anything starting with "ad" in the div (it was breaking videos when people had adblockers), not sure if they're smarter in general, but worth considering.  I assume in that comment of course that ad blockers wouldn't want to consider this as an advert, but maybe that's not the case!